images/Makefile

```diff
@@ -0,0 +1,10 @@
+HOOK_TAG := bandwagon-hook:$(VERSION)
+
+.PHONY: hook
+hook:
+	$(eval CHANGESET = $(shell echo $$VERSION | sed -e 's/[\.]//g'))
```

Comment:
nit: Since most of this target is already a (bash) script, I'd add the rest into the mix as well:
```Makefile
.PHONY: hook
hook:
	set -e ;\
	CHANGESET=$$(echo $(VERSION) | sed -e 's/[\.]//g') ;\
	if [ -z "$$CHANGESET" ]; then \
	  echo "VERSION is not set"; exit 1; \
	fi;\
	docker build --pull -t $(HOOK_TAG) \
		--build-arg CHANGESET=bandwagon-$$CHANGESET hook
```
this will make `CHANGESET` effectively private to `hook` (as otherwise is spills outside).